src/pages/index.module.css

```
@@ -3,7 +3,7 @@
  * and scoped locally.
  */
 
-.heroBanner {
+ .heroBanner {
```

```suggestion
.heroBanner {
```
why extra whitespace?